parlai/agents/transformer/polyencoder.py

```diff
@@ -96,6 +95,29 @@ def add_cmdline_args(cls, argparser):
         )
         return agent
 
+    @classmethod
+    def upgrade_opt(cls, opt_from_disk: Opt):
+        # call the parent upgrades
+        opt_from_disk = super(PolyencoderAgent, cls).upgrade_opt(opt_from_disk)
+
+        polyencoder_attention_keys_value = opt_from_disk.get(
+            'polyencoder_attention_keys'
+        )
+        if polyencoder_attention_keys_value is not None:
+            # 2020-02-19 We are deprecating this flag because it was used for a one-time
+            # set of experiments and won't be used again. This flag was defaulted to
+            # 'context', so throw an exception otherwise.
+            if polyencoder_attention_keys_value == 'context':
+                warn_once(
+                    '--polyencoder-attention-keys has been deprecated and will be ignored.'
+                )
+            else:
+                raise NotImplementedError(
+                    'This --polyencoder-attention-keys mode is no longer supported!'
```

Comment:
Suggest a commit where the user can find this option.

Reply:
Yes - added